composer.json

```diff
@@ -2,7 +2,7 @@
   "name": "elgentos/serversideanalytics2",
   "description": "Elgentos_ServerSideAnalytics for magento 2",
   "require": {
-    "php": ">=7.1.0",
+    "php": ">=8.1.0",
```

Comment:
is this really backwards incompatible?

Reply:
Yes. It already was, b/c of property definition in the constructor